### PR TITLE
[REF] web: ExportDialog revamp

### PR DESCRIPTION
**Pull request by xlu-odoo**
This commit simplifies the SCSS of the export dialog.

task-2816763

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by xlu-odoo** [comment]
@stefanorigano Could you please take a look? 🙏

File: addons/web/static/src/legacy/js/widgets/data_export.js [hunk 1]
```diff
@@ -265,7 +265,7 @@ var DataExport = Dialog.extend({
         this.$records = this.$('.o_export_tree_item');
         this.$records.each(function (i, el) {
             var $el = $(el);
-            $el.find('.o_tree_column').first().toggleClass('o_required', !!self.records[$el.data('id')].required);
+            $el.find('.o_tree_column').first().toggleClass('o_required border-bottom border-dark', !!self.records[$el.data('id')].required);
```
Comment (stefanorigano) [inline addons/web/static/src/legacy/js/widgets/data_export.js:268]: It doesn't achieve the desired result, the border seems a glitch :/

image

What about setting it to `font-weight-bolder` instead? 
image

**Review by stefanorigano — CHANGES_REQUESTED** [review]
Just one remark, for the rest is ok considering the task's scope*.
Thanks, once done you can remove the draft state and move to tech review.

*This component needs to be deeply refactored in order to improve consistency and design. Added to task-2818586

File: addons/web/static/src/legacy/js/widgets/data_export.js (hunk 1 above)
Comment (xlu-odoo) [reply addons/web/static/src/legacy/js/widgets/data_export.js:268]: I definitely like more the font-weight-bolder result. Will update.

**Comment by xlu-odoo** [comment]
@adr-odoo ready for tech review  👍

**Comment by stefanorigano** [comment]
@pparidans , thanks :)

**Review by pparidans — APPROVED** [review]
LGTM.

Just added the `flex-grow-1` class to the `o_right_field_panel` and `o_left_field_panel` to made them grow and match the other one's height (or fill the space on mobile).

@robodoo r+

**Pull request closed without merging**